bin/citadel

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright:: Copyright 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+function usage() {
+  cat /dev/stdin <<DOC
+Usage: ${0##*/} FILE
+
+A Bash utility that prints the contents of the given FILE to STDOUT.
```

Comment:
I think I was confused with the usage of this command, "Given File"? I think we should say that the file lives in an S3 bucket. But maybe that was just me. Not sure the use case for this command though so totally a nit.